File: 0389-find-the-difference/0389-find-the-difference.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        alpha = [0 for x in range(26)]
        for char in s:
            alpha[ord(char)-97]+=1
        for char in t:
            alpha[ord(char)-97]-=1
        for ele in range(len(alpha)):
            if alpha[ele] != 0:
                return chr(ele+97)

File: 0389-find-the-difference/0389-find-the-difference.java
class Solution {
    public char findTheDifference(String s, String t) {
	    int output = 0;
	    output ^= (int)t.charAt(t.length()-1);
	    for(int i=0;i<s.length();i++) {
	        output ^= (int)s.charAt(i);
	        output ^= (int)t.charAt(i);
	    }
	    return (char)output;
        
        // int[] alpha = new int[26];
	    // char res = Character.MIN_VALUE;
	    // for(int i=0;i<s.length();i++)
	    //     alpha[s.charAt(i)-'a']++;
	    // for(int i=0;i<t.length();i++)
	    //     alpha[t.charAt(i)-'a']--;
	    // for(int i=0; i<26;i++) {
	    // if(alpha[i]!=0) {
	    // res = (char)(i+'a');
	    // }
	    // }
	    // return res;
          
    }
}